packages/google_sign_in/google_sign_in/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * Updates support matrix in README to indicate that iOS 11 is no longer supported.
 * Clients on versions of Flutter that still support iOS 11 can continue to use this
   package with iOS 11, but will not receive any further updates to the iOS implementation.
+* Allow app to handle platform exception when user canceled sign in flow
```

Comment:
I'd rewrite the Change as "`signIn` method now throws an exception when the user cancels the sign-in flow, instead of failing silently.".

I'd probably consider this a breaking change, as it will cause errors to users who are not currently handling the error.